### PR TITLE
Bump nodejs-patternfly-react to 2.29.0

### PR DESCRIPTION
**Pull request by sharvit**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.20
* [ ] 1.19

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: packages/foreman/nodejs-patternfly-react/nodejs-patternfly-react.spec [hunk 1]
```diff
@@ -17,6 +18,8 @@ Requires: npm(classnames) >= 2.2.5
 Requires: npm(classnames) < 3.0.0
 Requires: npm(css-element-queries) >= 1.0.1
 Requires: npm(css-element-queries) < 2.0.0
+Requires: npm(lodash) >= 4.17.11
```
Comment (ekohl) [inline packages/foreman/nodejs-patternfly-react/nodejs-patternfly-react.spec]: Looks like we include 4.17.10 in our repo. Could you bump it as well? The same script can handle it. Can all be part of the same PR.

File: packages/foreman/nodejs-patternfly-react/nodejs-patternfly-react.spec [hunk 2]
```diff
@@ -1,10 +1,11 @@
 %global npm_name patternfly-react
+%global enable_tests 1
```
Comment (ekohl) [inline packages/foreman/nodejs-patternfly-react/nodejs-patternfly-react.spec]: We dropped this from the latest template. I've released 4.0.0 with it and should also be compatible with the latest nodejs. Do note that you may need to rebase your branch to include f4cb5706cb8838e73b8eb1d7b8b071e568b5c61e which makes it compatible.

File: packages/foreman/nodejs-patternfly-react/nodejs-patternfly-react.spec [hunk 3]
```diff
@@ -43,6 +46,8 @@ Requires: npm(sortabular) >= 1.5.1
 Requires: npm(sortabular) < 2.0.0
 Requires: npm(table-resolver) >= 3.2.0
 Requires: npm(table-resolver) < 4.0.0
+Requires: npm(uuid) >= 3.3.2
```
Comment (ekohl) [inline packages/foreman/nodejs-patternfly-react/nodejs-patternfly-react.spec]: We have 3.2.1. Could you update it as well?

**Comment by sharvit** [comment]
Thanks @ekohl, can you revisit please?

**Comment by ohadlevy** [comment]
lgtm

**Comment by ohadlevy** [comment]
@ekohl bump please?

**Pull request merged**